src/features/hotspots/root/HotspotsTicker.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useEffect, memo, useMemo } from 'react'
+import { useSelector } from 'react-redux'
+import TextTicker from 'react-native-text-ticker'
+import { BoxProps } from '@shopify/restyle'
+import { useTranslation } from 'react-i18next'
+import Box from '../../../components/Box'
+import {
+  fetchStats,
+  fetchCurrentOraclePrice,
+  fetchPredictedOraclePrice,
+} from '../../../store/helium/heliumDataSlice'
+import { RootState } from '../../../store/rootReducer'
+import { useAppDispatch } from '../../../store/store'
+import { useTextVariants } from '../../../theme/themeHooks'
+import { Theme } from '../../../theme/theme'
+import { locale } from '../../../utils/i18n'
+
+type Props = BoxProps<Theme>
+const HotspotsTicker = ({ ...boxProps }: Props) => {
+  const { t } = useTranslation()
+  const dispatch = useAppDispatch()
+  const { body2 } = useTextVariants()
+  const currentOraclePrice = useSelector(
+    (state: RootState) => state.heliumData.currentOraclePrice,
+  )
+  const hotspotCount = useSelector(
+    (state: RootState) => state.heliumData.hotspotCount,
+  )
+  const blockTime = useSelector(
+    (state: RootState) => state.heliumData.blockTime,
+  )
+
+  useEffect(() => {
+    dispatch(fetchCurrentOraclePrice())
+    dispatch(fetchPredictedOraclePrice())
+    dispatch(fetchStats())
+  }, [dispatch])
```

Comment:
Do we want to put this on a 5 min update interval? That is what the rest of the page is using for rewards info. If you check in `WelcomeOverview` there is a date that gets updated every 5 min, you could just pass it down and add it to the dependencies so this refreshes too?

Reply:
Or we could maybe make it like 15 or 30 minutes and do it within this component?

Reply:
Maybe I should do it every time it focuses?

Reply:
Hmm just dont want to hammer the API. I think every 5 min is enough

Reply:
What if I combine them both? On focus, if it's been more than 5 minutes.

Reply:
That sounds good